brightwind/load/station.py

```diff
@@ -821,6 +821,8 @@ def __get_properties(self):
                 sensors = _raise_child(meas_point, child_to_raise='sensor')
             else:
                 sensors = _raise_child(calib_raised, child_to_raise='sensor')
+                if sensors[0]['sensor_type_id'] == 'thermohygrometer': # because thermohygrometers possess multiple calibrations for different measurement types
```

Comment:
This bug fix should be generic and not specific to 'thermohygrometer', the bug was raised for this sensor_type but can happen for any other sensor_type that can measure more than one measurement_type. Probably bug should be fixed on  _raise_child function or somewhere else? Not sure where is this bug generated, please can you investigate?

On task 43 schema https://github.com/IEA-Task-43/digital_wra_data_standard/blob/master/schema/iea43_wra_data_model.schema.json#L415  there are other sensor_types that can measure multiple measurement_types, e.g `2d_ultrasonic`, `3d_ultrasonic` etc.